sdk/keyvault/Azure.Security.Keyvault.Keys/src/Azure.Security.Keyvault.Keys.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>This is the Microsoft Azure Key Vault Keys client library</Description>
+    <AssemblyTitle>Microsoft Azure.Security.KeyVault.Keys client library</AssemblyTitle>
+    <VersionPrefix>1.0.0</VersionPrefix>
+    <PackageTags>Microsoft Azure Key Vault Keys</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      ]]>
+    </PackageReleaseNotes>
+
+    <!-- Make sure that we don't pull in additional dependencies during build or package -->
+    <ImportDefaultReferences>false</ImportDefaultReferences>
```

Comment:
@weshaggard can we make this false by default for track 2 SDKs?

Reply:
Yes or we can update the default reference set to make sense for track 2 libraries.

Reply:
Don't worry about it as part of this PR. I'll clean these up as well as some of the other properties I want to centralize.

Reply:
Once your PR is merged I can clean up this project to follow the other ones

Reply:
I just merged it. Feel free to clean-up this project similar to what I did in the other projects.

Reply:
Will do